core/src/main/java/com/airbnb/aerosolve/core/transforms/StringTokenizerTransform.java

```diff
@@ -0,0 +1,48 @@
+package com.airbnb.aerosolve.core.transforms;
+
+import com.airbnb.aerosolve.core.FeatureVector;
+import com.airbnb.aerosolve.core.util.Util;
+
+import java.util.Collections;
+import java.util.Map;
+import java.util.Set;
+
+import com.typesafe.config.Config;
+
+/**
+ * Tokenizes strings
+ * "field1" specifies the key of the feature
+ * "field2" specifies the delimiter or regex used to tokenize
+ */
+public class StringTokenizerTransform extends Transform {
+  private String fieldName1;
+  private String delimiterOrRegex;
+  private String outputName;
+
+  @Override
+  public void configure(Config config, String key) {
+    fieldName1 = config.getString(key + ".field1");
+    delimiterOrRegex = config.getString(key + ".field2");
+    outputName = config.getString(key + ".output");
+  }
+
+  @Override
+  public void doTransform(FeatureVector featureVector) {
+    Map<String, Set<String>> stringFeatures = featureVector.getStringFeatures();
+    if (stringFeatures == null) {
+      return;
+    }
+
+    Set<String> feature1 = stringFeatures.get(fieldName1);
+    if (feature1 == null) {
+      return;
+    }
+
+    Set<String> output = Util.getOrCreateStringFeature(outputName, stringFeatures);
```

Comment:
You want a float feature here and output the sum of times the token occured.
e.g. "the cat sat on the mat" should have the : 2 cat : 1 sat : 1 on : 1 mat : 1